interpretation/ww_head.py

```diff
@@ -591,8 +591,11 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
                 # TODO
                 return todo()
             if is_command(message, ['fuck', 'hook', 'sleep'], True) and user_role == "Hooker":
-                # TODO
-                return todo()
+                msg = "**Usage:** Choose a player to sleep with, if that player is a wolf, you die. When you are sleeping with someone you are not at home, so you wont get killed.\n\n`" + prefix + "sleep <player>`\n\n"
```

Comment:
The description doesn't need to be that detailed. Saying you'll sleep with them is enough.